onnx/test/test_external_data.py

```diff
@@ -672,5 +674,28 @@ def test_check_model_absolute(self) -> None:
             checker.check_model(self.model_filename)
 
 
+class TestRuntimeQuantizationFilePathBug(TestLoadExternalDataBase):
+    def test_load_from_path_object(self) -> None:
+        model_path = Path(self.model_filename)
+        model = onnx.load_model(model_path)
+
+    @skip_if_no_onnxruntime
+    def test_file_path_is_correct_during_quantization(self) -> None:
+        from onnxruntime.quantization import quantize_dynamic, QuantType
+        
+        prev_dir = os.getcwd()
+        os.chdir(os.path.dirname(self.temp_dir))
+
+        try:
+            quantize_dynamic(
```

Comment:
Using `quantize_dynamic` here because that's where the initial issue occurred. `onnxruntime` has since patched this on their end by converting all `Path` objects to strings before making calls to `onnx`.